const.ts

```diff
@@ -0,0 +1,106 @@
+export const default_keywords = [
+    "and",
+    "del",
+    "from",
+    "not",
+    "while",
+    "as",
+    "elif",
+    "global",
+    "or",
+    "with",
+    "assert",
+    "else",
+    "if",
+    "pass",
+    "yield",
+    "break",
+    "except",
+    "import",
+    "print",
+    "class",
+    "exec",
+    "in",
+    "raise",
+    "continue",
+    "finally",
+    "is",
+    "return",
+    "def",
+    "for",
+    "lambda",
+    "try",
+    "int",
+    "bool",
+    "str",
+];
+
+export const default_functions = [
```

Comment:
Where does this list come from? do we need to check in with the built-in teams and somehow load this instead of hardcoding? just wondering

Reply:
this is the default built-in function from the python official website.